pages/docs/tutorials/getting-started/request_reply.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Request and reply pattern
+weight: 20
+---
+
+A quite common messaging pattern is [request-reply](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RequestReply.html), which describes a **requester**, which sends a request message and waits for a reply and a **replier** which receives the request and responds with a reply.
+
+As of AsyncPAI v3, this pattern can be described natively.
+ 
+# Describing a requester
+
+We are gonna use a very simple ping and pong example where a requestor sends the ping and responder respond with pong. Take notice this is an application-level ping and pong and is not related to ping/pong in standards such as WebSocket.
```

Comment:
```suggestion
"We are **gonna** ..."
```

**Please go through your document to correct any slang words you accidentally included.** "Gonna" is not appropriate for docs; slang is appropriate for blog posts or personal posts.

Reply:
Changed anything that poped out, knowing what is slang is almost impossible for me tbh. Cause "gonna" is not slang to me it's just more casual than `going to`. 

Do point out if you find anything else 😄